package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "install-ispc-action",
+  "version": "1.0.0",
+  "description": "Install ISPC GitHub action.",
+  "main": "src/main.js",
+  "type": "module",
+  "scripts": {
+    "run": "node src/main.js",
+    "build": "esbuild --bundle src/main.js --outfile=build/main.cjs --platform=node --format=cjs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ScatteredRay/install-ispc-action.git"
```

I guess we need to change `ScatteredRay`->`ispc` in this file.
```suggestion
    "url": "git+https://github.com/ispc/install-ispc-action.git"
```